src/styles/application/_errorScreen.scss

```diff
@@ -0,0 +1,42 @@
+.integr8ly-errorscreen {
+  &-backdrop {
+    position: fixed;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: 100%;
+    background-size: cover;
+    background-color: $pf-color-blue-600;
+    background-image: url("../../img/provisioning-bg.png");
+    z-index: 5;
+  }
+
+  &-error-img {
```

Comment:
Percentages should not be used to set margins. Using the proper template styling, this is not needed (see `loading.scss`.